README.md

```diff
@@ -334,6 +334,7 @@ You can set an environment variable `SKETCHLIB_INSTALL` to affect `python setup.
 - Empty: uses cmake
 - `conda`: sets library location to the conda environment, and uses `src/Makefile` (used to be used in conda-forge recipe)
 - `azure`: Uses `src/Makefile`
+- `jlees`: Uses `src/Makefile_fedora38`
 
```

Comment:
Worth giving this a more generic name in case it is helpful to others (or becomes the first in a series of such options)? Or is this definitely a one off?

Reply:
Yeah maybe `dev` or `local` would be better and less self aggrandising/deprecating (delete as appropriate)